docs/t-localpv.md

```diff
@@ -0,0 +1,96 @@
+---
+id: t-localpv
+title: Troubleshooting OpenEBS - Dynamic LocalPV
+sidebar_label: LocalPV
+---
+------
+
+<font size="5">General guidelines for troubleshooting</font>
+
+- Contact <a href="/docs/next/support.html" target="_blank">OpenEBS Community</a> for support.
+- Search for similar issues added in this troubleshootiung section.
+- Search for any reported issues on <a href=" https://stackoverflow.com/questions/tagged/openebs" target="_blank">StackOverflow under OpenEBS tag</a>
+
+<br>
+<hr>
+<br>
+
+[LocalPV PVC in Pending state](#pvc-in-pending-state)
```

Comment:
Should we add one more - explaining BDC in pending state - because there are no devices available on the selected node. as described in this comment:
https://github.com/openebs/openebs/issues/3312#issuecomment-745260953

(Could also go as a note into the uglocaldevice - in the pod creation step.)

Reply:
Made the changes in PR https://github.com/openebs/openebs-docs/pull/907